src/checker.rs

```diff
@@ -40,32 +42,51 @@ impl Checker {
             match res {
                 Ok(r) => {
                     let u = &site.url[..];
-                    match r.status().as_str() {
-                        "200" | "301" => {
+
+                    if self.allow_redirections && r.status().is_redirection() {
```

Comment:
careful you're basically no longer accepting return code 200 OK.

Reply:
I'd explicitly add ... or r.status().as_str() == "200" to this if statement.

Reply:
> careful you're basically no longer accepting return code 200 OK.

mmm, not true, why ?

Reply:
if it is not a redirection, or if a redirection is not allowed, it will match for the 200

Reply:
I guess I understand your point now.